spec/plurimath/integration/asciimath_spec.rb

```diff
@@ -248,7 +248,7 @@
 
       it 'returns instance of Asciimath' do
         expect{
-          convert(string_to_formula(string, :asciimath), :asciimath)
+          convert(string_to_formula(string, :latex), :latex)
```

Comment:
@suleman-uzair why is this value now `:latex` when it is expecting an AsciiMath instance?

Reply:
@ronaldtse 
The equations being processed in these specs are in **LaTeX**. but it was being parsed as **AsciiMath**, this change fixes it.
Currently, this file contains test cases for 2 different representations and I plan to create a new pull request (PR) to split them into different files according to their representations.

Reply:
I see, okay.